modules/get_me_some_ice_cream/__init__.py

```diff
@@ -0,0 +1,13 @@
+from get_me_some_ice_cream.states.detection import *
+from get_me_some_ice_cream.states.conversation_flow import *
+
+with rs.Module(name="Luigi") as mod:
```

Comment:
Shouldn't the python module also be called `luigi` instead of `get_me_some_ice_cream`?